src/main/scala/devices/debug/dm_registers.scala

```diff
@@ -797,13 +805,19 @@ class SBCSFields extends Bundle {
             While this field is non-zero, no more system bus accesses can be
             initiated by the debug module.
 
+            An implementation may report "Other" (7) for any error condition.
+
             0: There was no bus error.
 
             1: There was a timeout.
 
             2: A bad address was accessed.
 
-            3: There was some other error (eg. alignment).
+            3: There was an alignment error.
+
+            4: An access of unsupported size was requested.
```

Comment:
I'll need to make a change to my rocket-chip PR before submitting. Currently I don't differentiate between codes 3 and 4